src/main/treeDataProviders/dependenciesTree/dependenciesDataProvider.ts

```diff
@@ -67,7 +67,7 @@ export class DependenciesTreeDataProvider implements vscode.TreeDataProvider<Dep
         }
     }
 
-    getTreeItem(element: DependenciesTreeNode): vscode.TreeItem {
+    public getTreeItem(element: DependenciesTreeNode): vscode.TreeItem {
```

Comment:
```suggestion
    getTreeItem(element: DependenciesTreeNode): vscode.TreeItem {
```

Reply:
As we use strict mode as our "philosophy", why would we want to override it here?

Reply:
Okay